Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:latest
+
+RUN apt-get update
+RUN apt-get install npm  -y
+RUN npm install
+RUN go get github.com/go-sql-driver/mysql
+RUN go get github.com/DATA-DOG/go-sqlmock
+RUN go get github.com/stretchr/testify/assert
+RUN go get golang.org/x/crypto/bcrypt
```

Comment:
if you run `go get ./...` in the repo root, it will fetch all the necessary dependencies.
alternatively, use Go modules and vendor the dependencies instead of fetching while compiling.

Reply:
Vendoring in a requirement of this challenge and this doesn't really vendor these libraries. The current version that was fetched will be cached until changes are either made to the Dockerfile or a cache is wiped out.

You should use `go mod`.

Reply:
Ah, I missed that. I will fix that.